.github/workflows/static-test.yml

```diff
@@ -21,15 +21,15 @@ jobs:
         fetch-depth: 0
     - name: Setup git
       run: |
-        git fetch origin master:master_upstream --no-tags
+        git fetch origin ${{ github.base_ref }}:${{ github.base_ref }} --no-tags
         git config apply.whitespace nowarn
     - name: Fetch riot/riotbuild Docker image
       run: docker pull riot/riotbuild:latest
     - name: Run static-tests
       run: |
```

Comment:
unrelated: if you do "run: >", can't you save the "\"? yaml would take care of making it one line. https://yaml-multiline.info/

Reply:
Can try (outsude of this)... But I find it as it is currently less 'magic'. If for any reason there need to be more commands in this step, the contributor adding that command might get confused.

Reply:
Especially since `>` isn't documented in the GitHub Action documentation I believe.